package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "vitessce",
-  "version": "0.0.22",
+  "version": "0.0.20",
```

```suggestion
  "version": "0.0.23",
```
It looks like something regressed during a merge? I'd want to be extra sure that nothing else is moving backwards.

---

I just haven't updated to master in a while.